modules/040-node-manager/openapi/values.yaml

```diff
@@ -232,6 +234,9 @@ properties:
               # This is a copy from NodeGroup object. We trust that hook will not change it.
               type: object
               additionalProperties: true
+            nodeCapacity:
+              type: object
+              additionalProperties: true
```

Comment:
There can only be CPU and memory as far as I got it from the code. If so, let's describe them explicitly.

Reply:
It could be a `gpu` also in the future but I got your point